gr-blocks/lib/selector_impl.h

```diff
@@ -47,6 +47,8 @@ class selector_impl : public selector
     void forecast(int noutput_items, gr_vector_int& ninput_items_required);
     bool check_topology(int ninputs, int noutputs);
     void setup_rpc();
+    void handleMsgInputIndex(pmt::pmt_t msg);
```

Comment:
https://github.com/gnuradio/greps/blob/master/grep-0001-coding-guidelines.md#code-formatting-guidelines

No CamelCase please.

Reply:
LOL I:heart:CamelCase! Not a GR thing though 🥇